### DISCUSSION TITLE
The new environment variables are not working with pm2

### DISCUSSION THREAD

**Discussion by sarang-naico** [vercel/next.js#13758] [Help] [answered]
I have been using pm2 and I have configured environment variables in pm2 ecosystem.config.json.
The values are correctly loaded in the server.js but the next js is not passing it down to the browser.

sample ecosyste.config.js
`module.exports = {
    apps : [
        {
          name: "new-app",
          script: "./server.js",
          watch: true,
          env: {
            "NODE_ENV": "production",
            "NEXT_PUBLIC_TEST":"Testing"
          }
        }
    ]
  }`

**Comment by jamesmosier** [comment]
Next will only look for a `.env*` file according [to the docs](https://nextjs.org/docs/basic-features/environment-variables)

**Comment by timneutkens** [comment, marked as answer]
The variables exposed to the browser need to be known when `next build` is ran because they're inlined into the browser bundle.

**Reply by Vadorequest** [reply]
Exactly. @sarang-naico Take a look at this:

https://github.com/UnlyEd/next-right-now/blob/v2-mst-aptd-at-lcz-sty/next.config.js#L19-L34
